tests/tasks/test_unit_copy_hcs.py

```diff
@@ -56,24 +58,28 @@ def test_MIP(
     parallelization_list = copy_ome_zarr_hcs_plate(
         zarr_urls=zarr_urls,
         zarr_dir="tmp_out",
-        overwrite=True,
+        overwrite_images=True,
+        re_initialize_plate=True,
     )["parallelization_list"]
     debug(parallelization_list)
 
     # Run again, with overwrite=True
     parallelization_list_2 = copy_ome_zarr_hcs_plate(
         zarr_urls=zarr_urls,
         zarr_dir="tmp_out",
-        overwrite=True,
+        overwrite_images=True,
+        re_initialize_plate=True,
     )["parallelization_list"]
     assert parallelization_list_2 == parallelization_list
 
     # Run again, with overwrite=False
-    with pytest.raises(OverwriteNotAllowedError):
+    # TODO check with joel if this should be removed
```

Comment:
@lorenzocerrone This seems reasonable to keep. Any reason to remove it?

Reply:
I would keep it